### PR TITLE
Undefined workspace settings handling

### PR DESCRIPTION
**Pull request by olejorgenb**

### CODE REVIEW

File: settings.js [hunk 1]
```diff
@@ -133,6 +149,9 @@ function getNewWorkspaceSettings(index) {
     list.push(uuid);
     workspaceList.set_strv('list', list);
     settings.set_int('index', index);
+    settings.set_string('color', getUnusedColor());
+    let n = findWorkspaceSettingsByName(/^Unnamed .*/).length;
+    settings.set_string(`Unnamed ${n+1}`)
```
Comment (hedning) [inline settings.js]: Is there anything which prevents us from using `Workspace ${i}` here?

Comment (olejorgenb) [reply settings.js]: No, as long as the list is "monotonic" (which it is atm.) that would also work

Comment (hedning) [reply settings.js]: Right, I'd vote for `Workspace n` in that case, as that makes it clearer that it's a workspace related.

**Comment by jtaala** [comment]
Closing this as #584 addresses this issue - as well as work with dynamic workspaces (and injecting workspaces between other workspaces - e.g. non-monotonic workspace naming).

**Pull request closed without merging**